runtime/src/main/java/com/redhat/ipaas/rest/BaseHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.ipaas.rest;
+
+import javax.inject.Inject;
+
+public class BaseHandler implements WithDataManager {
+
+    @Inject
+    private DataManager dataMgr;
+
+    @Override
+    public DataManager dataManager() {
```

Comment:
There are tons of tools and frameworks that make use of the getter convention and so little gain.